doc/radosgw/notifications.rst

```diff
@@ -84,6 +84,18 @@ which tells the client that it may retry later.
 .. tip:: To minimize the latency added by asynchronous notification, we 
    recommended placing the "log" pool on fast media.
 
+The following are the global configurations for asynchronous notifications:
+
+.. confval:: rgw_bucket_persistent_notif_num_shards
+
+.. note:: For topics created during upgrade, per-key reordering of notifications may
```

Comment:
A topic might be created by a Ceph upgrade?  Or is this a timing thing?  Suggest:

.. note:: When a topic is created during a Ceph upgrade, per-key ..